src/sdltiles.cpp

```diff
@@ -1131,10 +1131,38 @@ int HandleDPad()
  * allows the input_manager to only consider those.
  * @return 0 if the input can not be translated (unknown key?),
  * -1 when a ALT+number sequence has been started,
- * or somthing that a call to ncurses getch would return.
+ * or something that a call to ncurses getch would return.
  */
 long sdl_keysym_to_curses(SDL_Keysym keysym)
 {
+
+    //Shift + Arrow (diagonal clockwise)
+    if( keysym.mod & KMOD_SHIFT ){
+        switch (keysym.sym) {
+            case SDLK_LEFT:
+                return '7'; //LEFTUP;
```

Comment:
Numeric constants in `src/input.h` would be better than the magic numbers.

Reply:
I agree. My final aim is to be able to load keys from keybindings.

Reply:
Movement keys are now taken from input manager.